src/storageManager.js

```diff
@@ -119,6 +121,10 @@ export function newStorageManager({gvlid, moduleName, bidderCode, moduleType} =
    * @returns {boolean}
    */
   const localStorageIsEnabled = function (done) {
+    const consentData = gdprDataHandler.getConsentData();
+    if (!consentData || !hasPurpose1Consent(consentData)) {
+      return false;
+    }
     let cb = function (result) {
       if (result && result.valid) {
```

Comment:
ok, no problem, I will close this PR and will open an issue, because prebid is setting "cookieTest" without regarding consent, what this PR should fix, but if this PR isn't good enough, please, go on clear the issue by yourself